jenkins-plugin/src/main/java/org/jenkinsci/plugins/pipeline/maven/WithMavenStepExecution.java

```diff
@@ -1044,7 +1050,7 @@ public void expand(EnvVars env) throws IOException, InterruptedException {
     /**
      * Callback to cleanup tmp script after finishing the job
      */
-    private static class WithMavenStepExecutionCallBack extends BodyExecutionCallback.TailCall {
+    private class WithMavenStepExecutionCallBack extends GeneralNonBlockingStepExecution.TailCall {

```

Comment:
or you can omit the `GeneralNonBlockingStepExecution.` part and it will resolve correctly